applications/serial_lte_modem/prj.conf

```diff
@@ -109,3 +109,8 @@ CONFIG_SLM_LOCATION=n
 # Serial DFU support
 CONFIG_SLM_NRF52_DFU=n
 CONFIG_SLM_NRF52_DFU_LEGACY=n
+
+#CONFIG_NRF_MODEM_LIB_TRACE=y
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_BACKEND_UART=n
+#CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART=y
```

Comment:
I'm not sure that these 5 lines should be added. The `CONFIG_LOG_BACKEND` settings are already on line 18 and 19, and the `LIB_TRACE` related ones are in the overlay.

Reply:
These were added when testing and should not be here :)